service/gcs/transport/mock.go

```diff
@@ -15,43 +19,54 @@ type MockTransport struct {
 
 // Dial ignores the port, and returns a MockTransport struct.
 func (t *MockTransport) Dial(_ uint32) (Connection, error) {
-	csr, csw := io.Pipe()
-	scr, scw := io.Pipe()
+	sockDir, err := ioutil.TempDir("", "mocktport")
```

Comment:
os.Pipe doesn't support the File() method. This is a problem for the runc wrapper code, which relies on this method existing so that it can dup the file descriptors into the runc process. Without this change, the runc tests fail if they try to do anything with stdio.